libraries/cfstore/scripts/cfstore_test_spec_k64f_gcc.json

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
@simonqhughes @0xc0170 You can use latest Greentea from PyPI. It allows you to select subset of tests using `-n`. For example:
```
$ mbedgt -VS -n mbed-os-core-configuration-store-TESTS-testgroup*
```
Note: Note `*` at the end of the name.

It will execute all tests starting with `mbed-os-core-configuration-store-TESTS-testgroup`.
See https://github.com/ARMmbed/greentea/releases/tag/v0.2.16 and https://github.com/ARMmbed/greentea/pull/137 for details!